source/includes/_02-passfort-data-structure.html.md

```diff
@@ -1,4 +1,3 @@
-
 # Data Structure
 
 PassFort structures data on individuals and company entities in a standard
```

Comment:
"individual and company entities" (no "s")

Reply:
"Additionally, many data structure types are common across different checks." 

(reword + full stop)